conf/airframes/AGGIEAIR/aggieair_rp3_lia.xml

```diff
@@ -35,6 +35,8 @@ RP3 Lisa MX
 
     <define name="AGR_CLIMB" />
     <define name="POLY_OSAM_HALF_SWEEP_ENABLED" value="FALSE"/>
+    <define name="AIRSPACE"/>
+    <define name="DATALINK_LOST_TIME" value="45"/>
```

Comment:
Even if "controlled" from the airframe file, it is still a setting that controls when the aircraft will enter HOME mode, so HOME_MODE_DATALINK_LOST_TIME is more appropriate I think.
Although it is nice to make it configurable from an airframe independently of the flight plan, I think it can also be useful to be able to specify it in a flight plan, so it works even if not defined in airframe. If nothing defined -> no check, if defined in airframe and/or FP, use FP unless airframe already defines it.
What do you think ?